### PR TITLE
chore: add redirections to centradev

### PR DESCRIPTION
**Pull request by mateusz-001**
#### Changelog entry
- Added redirections from docs.centra.com to new dev site - centra.dev (Adreas prepared [spreadsheet with all redirects](https://docs.google.com/spreadsheets/d/1x8zc8MHvddvXUrRBFzgNSuMIdHcdmKpreeqRc0X7tiY/edit?usp=sharing))

### CODE REVIEW

File: config/site.yaml [hunk 1]
```diff
@@ -43,3 +43,122 @@ redirects:
   /guides/erp-with-gql: '/guides/pim-gql-integration'
   /plugins/%20post-purchase-upsell: '/fe-development/cross-sell'
   /plugins/post-purchase-upsell: '/fe-development/cross-sell'
+  /overview: 'https://centra.dev/docs'
+  /overview/glossary: 'https://centra.dev/docs/centra-concepts/reference/centra-a-z'
+  /overview/centra-overview: 'https://centra.dev/docs/centra-concepts'
+  /overview/products: 'https://centra.dev/docs/centra-concepts/product'
```
Comment (remziatay) [inline config/site.yaml:49]: Will these handle trailing slash cases? e.g. `docs.centra.com/overview/`

File: config/site.yaml [hunk 2]
```diff
@@ -43,3 +43,122 @@ redirects:
...
+  /api-references/order-api/api-reference/get-supplier-delivery-details: 'https://centra.dev/docs/apis/order-api/api-reference'
+  /api-references/order-api/api-reference/accept-supplier-delivery: 'https://centra.dev/docs/apis/order-api/api-reference'
+  /api-references/order-api/api-reference/create-voucher: 'https://centra.dev/docs/apis/order-api/api-reference'
+  /api-references/order-api/api-reference/clone-voucher: 'https://centra.dev/docs/apis/order-api/api-reference'
+  /api-references/order-api/api-reference/get-voucher: 'https://centra.dev/docs/apis/order-api/api-reference'
+  /api-references/soap-integration-api: 'https://centra.dev/docs/apis/soap-integration-api/what-you-can-do-with-it'
+  /api-references/soap-integration-api/api-reference: 'https://centra.dev/docs/apis/soap-integration-api/api-reference'
+  /silksoap: 'https://centra.dev/docs/apis/soap-integration-api/api-reference'
+  /graphql-explorer: 'https://docs.centra.com/graphql-explorer'
+  /graphql: 'https://centra.dev/graphql'
```
Comment (remziatay) [inline config/site.yaml:162]: How about all the subpaths? (e.g. https://docs.centra.com/graphql/voucheractioncriteria.html)
Will this redirect them as well?

If not, it looks like regexes are supported https://learn.getgrav.org/17/content/routing#redirects

File: config/site.yaml [hunk 3]
```diff
@@ -43,3 +43,122 @@ redirects:
...
+  /swagger-ui/?urls.primaryName=ShopAPI: 'https://centra.dev/swagger?urls.primaryName=ShopAPI'
+  /swagger-ui/?urls.primaryName=CheckoutAPI: 'https://centra.dev/swagger?urls.primaryName=CheckoutAPI'
```
Comment (remziatay) [inline config/site.yaml:164]: `/swagger-ui` alone is also valid, can we handle that as well?

File: config/site.yaml [hunk 4]
```diff
@@ -43,3 +43,122 @@ redirects:
...
+  /api-references/shop-api/api-reference: 'https://centra.dev/docs/apis/shop-api/api-reference'
+  /api-references/shop-api/api-reference/get_brands: 'https://centra.dev/docs/apis/shop-api/api-reference'
+  /api-references/shop-api/api-reference/get-campaign-sites: 'https://centra.dev/docs/apis/shop-api/api-reference'
+  /api-references/shop-api/api-reference/get-campaigns: 'https://centra.dev/docs/apis/shop-api/api-reference'
+  /api-references/shop-api/api-reference/get-categories: 'https://centra.dev/docs/apis/shop-api/api-reference'
+  /api-references/shop-api/api-reference/get-collections: 'https://centra.dev/docs/apis/shop-api/api-reference'
+  /api-references/shop-api/api-reference/get-measurement-charts: 'https://centra.dev/docs/apis/shop-api/api-reference'
+  /api-references/shop-api/api-reference/get-product-ids: 'https://centra.dev/docs/apis/shop-api/api-reference'
+  /api-references/shop-api/api-reference/get-products: 'https://centra.dev/docs/apis/shop-api/api-reference'
+  /api-references/shop-api/api-reference/get-products-filtered: 'https://centra.dev/docs/apis/shop-api/api-reference'
```
Comment (remziatay) [inline config/site.yaml:121]: Let's use regex when they all redirect to same place (didn't test)
```yaml
/api-references/shop-api/api-reference/.*: 'https://centra.dev/docs/apis/shop-api/api-reference'
```

**Comment by remziatay** [comment]
Not related but trailing slashes currently redirect to `centra-api-documentation.herokuapp.com`. What's up with that?
e.g.
https://docs.centra.com/overview/ redirects to https://centra-api-documentation.herokuapp.com/overview

File: config/site.yaml (hunk 3 above)
Comment (mateusz-001) [reply config/site.yaml:164]: Yeah, will add it.

File: config/site.yaml (hunk 1 above)
Comment (mateusz-001) [reply config/site.yaml:49]: Yes, it will handle trailing slashes ;)

File: config/site.yaml (hunk 4 above)
Comment (mateusz-001) [reply config/site.yaml:121]: Works! Thanks for this 😄

**Comment by mateusz-001** [comment]
> Not related but trailing slashes currently redirect to `centra-api-documentation.herokuapp.com`. What's up with that? e.g. https://docs.centra.com/overview/ redirects to https://centra-api-documentation.herokuapp.com/overview

Honestly a have no idea why it's happening, there isn't any redirects related with `https://docs.centra.com/overview/` 🤔

**Comment by remziatay** [comment]
> Honestly a have no idea why it's happening, there isn't any redirects related with `https://docs.centra.com/overview/` 🤔

Very likely https://github.com/centrahq/api-documentation-proxy is doing this

**Review by remziatay — APPROVED** [review]

**Pull request merged**